core/kubernetes/openhim/openhim-core-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: openhim-core-service
+spec:
+  type: LoadBalancer
+  selector:
+    component: openhim-core
+  ports:
+    - port: 8082
+      name: core-8080
+      targetPort: 8080
+    - port: 5000
+      name: core-5000
+      targetPort: 5000
+    - port: 5001
+      name: core-5001
+      targetPort: 5001
```

Comment:
minor: Maybe we can re-order the items to first define the name, and then the port/targetPort